lib/oembed/utils.py

```diff
@@ -7,12 +7,16 @@
 from htk.lib.oembed.constants import *
 from htk.utils.request import get_current_request
 
-def get_oembed_html(url):
+def get_oembed_html(url, autoplay=False):
     """Gets the oEmbed HTML for a URL, if it is an oEmbed type
     """
     oembed_type = get_oembed_type(url)
+
     if oembed_type:
-        html = get_oembed_html_for_service(url, oembed_type)
+        if oembed_type == "youtube":
```

Comment:
@jontsai Do you prefer ternary operator here? It would be a long like.
`html = youtube_oembed(url, autoplay) if oembed_type == 'youtube' else get_oembed_html_for_service(url, oembed_type)`

Reply:
I like how you have it

Reply:
how you currently have it is most readable

Reply:
it's like riding bike against traffic (wrong) vs with traffic (right) -- driver's aren't trained to dodge bicyclists, but to go around them

in the same way, devs are used to scrolling vertically, not horizontally / having their eyes wrap at lines

Reply:
Will keep in mind.